docs/7 Frontend/1 Overview.md

```diff
@@ -0,0 +1,19 @@
+# Frontend Overview
+
+When scaffolding a blockchain, Starport also creates a Vue frontend application in `vue` directory. If this directory exists, `starport serve` automatically runs `npm run serve`.
+
+The frontend app is built using the `@starport/vue` and `@starport/vuex` packages. More information and documentation can be found in the [`tendermint/vue`](https://github.com/tendermint/vue) monorepo.
+
+## Client code generation
+
+Starport automatically generates JavaScript, TypeScript and Vuex clients for your blockchain both for custom and standard Cosmos SDK modules. To enable client code generation add the following to `config.yml`:
+
+```yaml
+client:
+  vuex:
+    path: "js"
+```
+
+Starport will generate a Vuex client inside `js` directory. JS and TS clients are also generated, because they are dependencies of the Vuex client.
```

Comment:
```suggestion
A Vuex client is generated in the `js` directory. JS and TS clients are also generated because they are dependencies of the Vuex client.
```